### DISCUSSION TITLE
Vercel deployment is not happening even though git hub is updated. Please help

### DISCUSSION THREAD

**Discussion by surajnarayanan1** [vercel/community#3631] [Help] [answered]
### Summary

Vercel deployment is not happening even though git hub is updated. Please help

### Steps to Reproduce

Vercel deployment is not happening even though git hub is updated. Please help

**Comment by amyegan** [comment, marked as answer]
Hey @surajnarayanan1. Sorry about the deployment issues. There is an ongoing GitHub incident.

> Due to a problem with GitHub, we are currently experiencing delayed deployments made with Vercel for GitHub. For more information, see the GitHub status: https://www.githubstatus.com/incidents/ljhhhw2tplqp
> 
> In the meantime, you can still use the Vercel CLI to deploy: https://vercel.com/docs/cli
